doc/project/checklist.md

```diff
@@ -321,18 +352,31 @@ Ensure that the complete testbench code is free from TODOs.
 
 X Propagation Analysis complete
 
-### NIGHTLY_REGRESSION_AT_100
+### SIM_NIGHTLY_REGRESSION_AT_100
```

Comment:
Make these '_V3' to match their V2 counterparts?

Reply:
sure, updated!